content/backstage/plugins/incident-io.md

```diff
@@ -0,0 +1,70 @@
+---
+humanName: Incident.io
+heading: 'Backstage Incident.io Plugin'
+lead: 'Display incidents associated to Backstage components'
+attribution:
+  text: Incident.io
+  href: https://incident.io/
+
+intro: |
+  <p>
+    As a prerequisite, you need to have configured the Incident.io catalog-importer to sync with your Backstage catalog. You can visit your incident.io dashboard to create a custom field that is powered by the Backstage Component catalog type.
```

Comment:
Can we link to the "catalog-importer", whatever that is?

Reply:
I see it's linked below, maybe worth wrapping the relevant words in this text into <a> also

Reply:
Makes sense. Done